Update/omot_012.txt

```diff
@@ -2729,10 +2729,10 @@ void main()
 			NULL, "\"Don't take entrance exams lightly, all right?", Line_WaitForInput);
 	ModPlayVoiceLS(4, 1, "ps3/s14/01/210101157", 256, TRUE);
 	OutputLine(NULL, "　ポイント絞ってやらなきゃ下手すりゃ全教科書けませんでした、無条件降伏。",
-			NULL, " You have to squeeze the points out of your textbooks, on every single subject, otherwise you're just surrendering unconditionally.", Line_WaitForInput);
+			NULL, " She couldn't hardly write answers in any and every subject, she just surrendered unconditionally, so she should have hammered away  selectively.", Line_WaitForInput);
```

Comment:
```suggestion
			NULL, " You have to squeeze the points out of your textbooks, on every single subject, otherwise you're just surrendering unconditionally.", Line_WaitForInput);
```